debian/ocf-atool.postinst

```diff
@@ -0,0 +1,18 @@
+# create conf directory
+if [ ! -e /etc/ocf-atool ]; then
```

Comment:
already did in b89a04607a9faefb55a20dd97eec2e3ec9ede17e except for moving DEBHELPER